vhdbuilder/scripts/automate_version_bump.sh

```diff
@@ -40,10 +40,12 @@ create_image_bump_pr() {
 
 # This function cuts the official branch based off the commit ID that the builds were triggered from and tags it
 cut_official_branch() {
-    # official/vYYYYMMDD
-    official_branch_name="$(echo -n "official/v${new_image_version}" | head -c-1)"
-    # YYYYMM.DD.$REVISION
-    official_tag="v${new_image_version}"
+    # Image version format: YYYYMM.DD.revision
```

Comment:
nit: revision and patch here are the the same thing

Reply:
let's make the comments consistent to avoid confustion